tests/ipex/test_modeling.py

```diff
@@ -66,7 +66,7 @@
     "gpt_neox": "hf-internal-testing/tiny-random-GPTNeoXForCausalLM",
     "gptj": "hf-internal-testing/tiny-random-GPTJModel",
     "levit": "hf-internal-testing/tiny-random-LevitModel",
-    "llama": "fxmarty/tiny-llama-fast-tokenizer",
+    "llama": "Jiqing/tiny_random_llama2",
```

Comment:
would be in favor of keeping the current one and adding an other test instead
```suggestion
    "llama": "fxmarty/tiny-llama-fast-tokenizer",
    "llama2": "Jiqing/tiny_random_llama2",
```

Reply:
also to fix the test we would need to update the tokenizer from https://huggingface.co/Jiqing/tiny_random_llama2/